### PR TITLE
Reshape

### PR DESCRIPTION
**Pull request by tiffanychu90**
* Fix script to download Census API correctly if ACS variables change over time. Occurs if the ACS table gets larger, the same variables we're interested in might shift in place.
* Import various boundary files, clip to City of LA boundary, and project
* Create crosswalk files that link tracts to zipcodes, neighborhoods, CDs, etc. Will need for aggregating up to larger geographies.
* Start cleaning values (i.e percents are 0-1 rather than 0-100)

### CODE REVIEW

File: src/B2_create_boundary_crosswalks.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import numpy as np
+import pandas as pd
+import geopandas as gpd
+import intake
+import os
+import utils
+
+catalog = intake.open_catalog('./catalogs/*.yml')
+
+tracts = gpd.read_file('./gis/raw/tracts_clipped.geojson')
```
Comment (ian-r-rose) [inline src/B2_create_boundary_crosswalks.py]: Note to @ian-r-rose: investigate persistence in intake.

Comment (tiffanychu90) [reply src/B2_create_boundary_crosswalks.py]: Resolved in issue #3

File: Makefile [hunk 2]
```diff
@@ -12,3 +12,6 @@ install: conda pip
 mirror:
 	intake-dcat mirror manifest.yml > catalogs/open-data.yml
 
+#census_data:
```
Comment (hunterowens) [inline Makefile]: any reason this is commented out

Comment (tiffanychu90) [reply Makefile]: It gets added back in later. Wasn't sure how to call the R script in a chain.

**Pull request merged**